### DISCUSSION TITLE
visual bell when tmux changing window on remote terminal

### DISCUSSION THREAD

**Discussion by danielb2** [kovidgoyal/kitty#6884] [General]
recently Kitty (or something) is triggering a visual bell everytime (or almost) I switch videos using tmux. This is so annoying I am considering disabling the visual bell,  but I think VB has it's uses too. Any idea what could be causing this? 

This isn't kitty specific, it's also happening in iTerm. I'm just hoping someone can help

**Comment by danielb2** [comment]
I'll close for now. I'll ask on tmux first

**Comment by kovidgoyal** [comment]
I have no idea what switching videos using tmux means. tmus doesnt play
videos. And the visual_bell is triggerred in kitty when the program
runinng in it prints the bell character, that is when a normal bell
occurrs and you have setup kitty to do a visual bell. So figure out
which of your programs is printing the nell character.

**Reply by danielb2** [reply]
Yeah I can't figure it out. I can't reproduce it locally and for example I'm running weechat in a window and switching to that window will even trigger it . And switching windows means, for example `ctrl + b a`
